frontend/src/index.js

```diff
@@ -32,6 +37,31 @@ const router = createBrowserRouter([
         element: <SingleUserCollectionsPage />,
         errorElement: <ErrorPage />,
       },
+      {
+        path: 'login',
+        element: <LoginPage />,
+        errorElement: <ErrorPage />,
+      },
+      {
+        path: 'signup',
+        element: <SignupPage />,
+        errorElement: <ErrorPage />,
+      },
+      {
+        path: 'user-profile',
+        element: <UserProfilePage />,
+        errorElement: <ErrorPage />,
+      },
+      {
+        path: 'Password1',
```

Comment:
why is the url path like `/Password1` and `/Password2`? Please change to something meaningful like `/reset`